src/main/java/seedu/address/ui/FullPersonCard.java

```diff
@@ -64,6 +64,11 @@ public FullPersonCard(ObservableList<Person> persons, double windowWidth) {
 
         fullPersonDetails.setPrefWidth(windowWidth / 3);
         //cardPane.setPrefWidth(windowWidth / 3);
+
+        if (displayedIndex == 0) {
+            createEmptyCard();
```

Comment:
Is there a need for this? Its already caught in the indexOutOfBounds catch block

Reply:
oh yea good point, will change